devops/openshift/init-secrets.yml

```diff
@@ -165,6 +166,11 @@ parameters:
     required: true
     description: |
       Folder on the SFTP where SFAS integration files will be placed.
+  - name: IER_REQUEST_FOLDER
+    displayName: IER request folder
+    required: true
+    description: |
+      Folder on the SFTP where Institution integration files will be placed.
```

Comment:
Is it the folder for all institutions or only for IER12?
Do we nee a specific folder for IER or only a generic one for any Institution Integration?